src/app/routes/api/v3/admin/forms/admin-forms.settings.routes.ts

```diff
@@ -24,8 +24,9 @@ const updateSettingsValidator = celebrate({
     submissionLimit: Joi.number().allow(null),
     title: Joi.string(),
     webhook: Joi.object({
-      url: Joi.string().uri().required().allow(''),
-    }),
+      url: Joi.string().uri().allow(''),
+      isRetryEnabled: Joi.boolean(),
+    }).min(1),
```

Comment:
do we need `min` for webhooks? should it be more specific? no `url` means no `isRetryEnabled`?

EDIT: I see both have defaults, so i don't think `min` is needed

Reply:
maybe it's for subsequent updates? so that at least one of the two keys are set on subsequent updates if they do want to update webhook config

Reply:
> EDIT: I see both have defaults, so i don't think min is needed

haha but if the frontend sends the update `webhook: {}`, what does that mean? what should be the expected behaviour? and how does mongoose even behave when we do `findByIdAndUpdate(formId, { webhook: {} })`? does that conform to the behaviour expected when the frontend sends this update?

to be honest I don't really want to find out any of the answers, which is why the `.min(1)` is there hahahaha

Reply:
yea, requiring at least one sub-key if `webhooks` object is sent to the backend makes sense oki